buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java

```diff
@@ -46,19 +46,20 @@
  */
 public abstract class AbstractByteBuf extends ByteBuf {
     private static final InternalLogger logger = InternalLoggerFactory.getInstance(AbstractByteBuf.class);
-    private static final String LEGACY_PROP_CHECK_ACCESSIBLE = "io.netty.buffer.bytebuf.checkAccessible";
-    private static final String PROP_CHECK_ACCESSIBLE = "io.netty.buffer.checkAccessible";
+    private static final String LEGACY_PROP_CHECK_ACCESSIBLE = "siren.io.netty.buffer.bytebuf.checkAccessible";
+    private static final String PROP_CHECK_ACCESSIBLE = "siren.io.netty.buffer.checkAccessible";
     static final boolean checkAccessible; // accessed from CompositeByteBuf
-    private static final String PROP_CHECK_BOUNDS = "io.netty.buffer.checkBounds";
+    private static final String PROP_CHECK_BOUNDS = "siren.io.netty.buffer.checkBounds";
     private static final boolean checkBounds;
 
     static {
+	// siren: set the default value to false
         if (SystemPropertyUtil.contains(PROP_CHECK_ACCESSIBLE)) {
             checkAccessible = SystemPropertyUtil.getBoolean(PROP_CHECK_ACCESSIBLE, true);
```

Comment:
this should be false too

Reply:
thanks, done https://github.com/sirensolutions/netty/pull/17/commits/655b0ea679ef84e382b2e3f08106adace0643f28